.github/release-drafter.yml

```diff
@@ -1,34 +1,36 @@
 branches:
   - master
 
-name-template: 'Version $NEXT_PATCH_VERSION'
+name-template: 'Zeus Enhanced $NEXT_PATCH_VERSION'
 tag-template: 'v$NEXT_PATCH_VERSION'
 
 categories:
   - title: '**ADDED:**'
     labels: 'feature'
-  - title: '**FIXED:**'
-    labels: 'bug fix'
   - title: '**IMPROVED**'
     labels:
       - 'enhancement'
       - 'optimization'
   - title: '**CHANGED**'
     labels: 'cleanup'
+  - title: '**FIXED:**'
+    labels: 'bug fix'
   - title: '**TRANSLATIONS**'
     labels: 'translation'
 
 exclude-labels: 'ignore changelog'
 
 change-template: '- $TITLE ($NUMBER)'
 template: |
-  _Requires CBA version X.Y.Z or later and Arma 3 version X.Y or later._
+  _Requires CBA Version X.Y.Z or later and Arma 3 Version X.Y or later._
+
+  DESCRIPTION HERE
 
-  ## Changelog Summary
+  ## Change Log Summary
```

Comment:
Changelog is actually the spelling used on wiki and in many other places I encounter. I don't see Change Log being used as much.

Reply:
It has been used in all releases so far.